### PR TITLE
Javadoc Fixes

### PR DESCRIPTION
**Pull request by aerouk**
- formatting
- added some descriptors for classes
- fixed some spacing and some whitespaces

lmk if I did anything wrong, thanks.

### CODE REVIEW

File: src/main/java/org/bukkit/event/entity/EntityChangeBlockEvent.java [hunk 1]
```diff
@@ -18,7 +18,6 @@
     private final byte data;
 
     /**
-     *
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/event/entity/EntityChangeBlockEvent.java:21]: I don't care for this particular removal, nor the next one. It should really have a description, retaining this line break.

File: src/main/java/org/bukkit/event/player/AsyncPlayerChatEvent.java [hunk 2]
```diff
@@ -25,7 +25,6 @@
     private final Set<Player> recipients;
 
     /**
-     *
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/event/player/AsyncPlayerChatEvent.java:28]: https://github.com/Bukkit/Bukkit-JavaDoc/pull/34/files#pullrequestreviewcomment-5486152

**Comment by aerouk** [comment]
Pushing a fix now

File: src/main/java/org/bukkit/event/entity/EntityChangeBlockEvent.java [hunk 3]
```diff
@@ -30,7 +30,6 @@ public EntityChangeBlockEvent(final LivingEntity what, final Block block, final
     }
 
     /**
-     *
```
Comment (Wolvereness) [inline src/main/java/org/bukkit/event/entity/EntityChangeBlockEvent.java:33]: This one was missed.

**Comment by aerouk** [comment]
Sorry, pushing a fix for that now too.

**Pull request merged**